avm/res/net-app/net-app-account/snapshot-policies/main.bicep

```diff
@@ -110,3 +50,92 @@ output name string = snapshotPolicies.name
 
 @description('The name of the Resource Group the Snapshot was created in.')
 output resourceGroupName string = resourceGroup().name
+
+// ================ //
+// Definitions      //
+// ================ //
+
+@export()
+type dailyScheduleType = {
```

Comment:
Also here missing UDT description decorator

Reply:
added to all